src/Http/Controllers/CodelessController.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace ShopEngine\Nova\Http\Controllers;
+
+use Illuminate\Http\Request;
+use Laravel\Nova\Http\Requests\NovaRequest;
+
+class CodelessController extends ShopEngineNovaController
+{
+    public function toggleStatus(Request $request)
+    {
+        return $this->getClient()->post('codeless/toggle-status', $request->codeless);
```

Comment:
This indicates you are passing an entire model as a parameter. Name the variables to what they are exactly, In this case it is the codeless' aggregate id.